opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -152,3 +156,123 @@ def _python_path_without_directory(python_path, directory, path_separator):
         "",
         python_path,
     )
+
+
+_OTEL_SEMCONV_STABILITY_OPT_IN_KEY = "OTEL_SEMCONV_STABILITY_OPT_IN"
+
+# Old to new http semantic convention mappings
+_OTEL_HTTP_SEMCONV_MIGRATION_MAPPING = {
+    SpanAttributes.HTTP_METHOD: SpanAttributes.HTTP_REQUEST_METHOD,
+    SpanAttributes.HTTP_STATUS_CODE: SpanAttributes.HTTP_RESPONSE_STATUS_CODE,
+    SpanAttributes.HTTP_SCHEME: SpanAttributes.URL_SCHEME,
+    SpanAttributes.HTTP_URL: SpanAttributes.URL_FULL,
+    SpanAttributes.HTTP_REQUEST_CONTENT_LENGTH: SpanAttributes.HTTP_REQUEST_BODY_SIZE,
+    SpanAttributes.HTTP_RESPONSE_CONTENT_LENGTH: SpanAttributes.HTTP_RESPONSE_BODY_SIZE,
+    SpanAttributes.NET_HOST_NAME: SpanAttributes.SERVER_ADDRESS,
+    SpanAttributes.NET_HOST_PORT: SpanAttributes.SERVER_PORT,
+    SpanAttributes.NET_SOCK_HOST_ADDR: SpanAttributes.SERVER_SOCKET_ADDRESS,
+    SpanAttributes.NET_SOCK_HOST_PORT: SpanAttributes.SERVER_SOCKET_PORT,
+    SpanAttributes.NET_TRANSPORT: SpanAttributes.NETWORK_TRANSPORT,
+    SpanAttributes.NET_PROTOCOL_NAME: SpanAttributes.NETWORK_PROTOCOL_NAME,
+    SpanAttributes.NET_PROTOCOL_VERSION: SpanAttributes.NETWORK_PROTOCOL_VERSION,
+    SpanAttributes.NET_SOCK_FAMILY: SpanAttributes.NETWORK_TYPE,
+    SpanAttributes.NET_PEER_IP: SpanAttributes.CLIENT_SOCKET_ADDRESS,
+    SpanAttributes.NET_HOST_IP: SpanAttributes.SERVER_SOCKET_ADDRESS,
```

Comment:
should map to `network.local.address` (not in the released semconv yet)

Reply:
Will do.

Reply:
Done in latest push